docker/pypi/wmagent/wmagent-docker-run.sh

```diff
@@ -55,37 +54,55 @@ HOST_MOUNT_DIR=/data/dockerMount
 [[ -h /data/srv/wmagent ]] && rm -f /data/srv/wmagent
 ln -s $HOST_MOUNT_DIR/srv/wmagent /data/srv/wmagent
 
-
+# create the passwd and group mount point dynamically at runtime
+passwdEntry=$(getent passwd $wmaUser | awk -F : -v wmaHome="/home/$wmaUser" '{print $1 ":" $2 ":" $3 ":" $4 ":" $5 ":" wmaHome ":" $7}')
+groupEntry=$(getent group $wmaGroup)
+
+# workaround case where Unix account is not in the local system (e.g. sssd)
+[[ -d $HOST_MOUNT_DIR/etc/ ]] || (mkdir -p $HOST_MOUNT_DIR/etc) || exit $?
+if ! [ -f $HOST_MOUNT_DIR/etc/passwd ]; then
```

Comment:
same comment as all over the place when it comes to single  conditioned  `if` expressions with extra negation and heavily overloaded syntax.